core/ip/mac.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ip
+
+import (
+	"net"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// GetMACAddressForIP finds MAC address
+func GetMACAddressForIP(ip string) (string, error) {
+	var currentNetworkHardwareName string
+	interfaces, _ := net.Interfaces()
```

Comment:
Let's not ignore this error, It could happen if it exists. 
```suggestion
	interfaces, _ := net.Interfaces()
```